src/plugins/intel_cpu/src/extension.cpp

```diff
@@ -26,8 +26,37 @@
 #include "transformations/snippets/x64/op/perf_count_rdtsc.hpp"
 #include "transformations/snippets/x64/op/store_convert.hpp"
 
+namespace {
+
+template <typename Op>
+class TypeRelaxedExtension : public ov::OpExtension<ov::op::TypeRelaxed<Op>> {
```

Comment:
do we really need this class?
As I can see, IR deserializer uses "version" from serialized layers to find necessary extensions
https://github.com/openvinotoolkit/openvino/blob/ce7dcb00b38172268826102e5bca41d0922c0199/src/frontends/ir/src/ir_deserializer.cpp#L816

perhaps it's enough to fix Serialize only

Reply:
discussed offline